src/config.xml

```diff
@@ -276,6 +276,17 @@ Go to the <a href="commands.html">next</a> section or return to the
  Enabling this option can be useful when feeding doxygen a huge amount of source
  files, where putting all generated files in the same directory would otherwise
  causes performance problems for the file system.
+]]>
+      </docs>
+    </option>
+    <option type='bool' id='VERBATIM_LISTINGS' defval='0'>
+      <docs>
+<![CDATA[
+ If the \c VERBATIM_LISTINGS tag is set to \c YES then doxygen will uses
+ \begin{DoxyVerbatimCode} \end{DoxyVerbatimCode} LaTeX commands for enclosing source code.
+ Source code is provided as is (verbatim).
+ When it is set to \c NO then doxygen uses \begin{DoxyCode}. The source code is processed and 
+ provided as LaTeX commands
```

Comment:
there are here some terms (like `\begin{DoxyVerbatimCode}`) that are not understood by a normal user but are internal LaTeX codes and should not be used in the documentation.

Reply:
I will try to rephrase it, and I see that `\` char fails doc generation for `doxygen` anyway.

Reply:
As a side note: putting the `\end{DoxyVerbatimCode}` between backticks (`` ` ``) would most likely already solve the warning, but the "not understood" will still remain of course.